### ISSUE TITLE
`python` system version (for `python3`) not available in Ubuntu 17.10

### ISSUE THREAD

**Issue by bittner** [pyenv/pyenv#1029]
I've set up a clean Ubuntu 17.10 Artful developer machine, which ships only a [(minimal) Python 3](https://packages.ubuntu.com/artful/python3-minimal) installation. In the Terminal you can now only run Python via the `python3` command. No `python` command or alias is provided out of the box.

I have now installed a few Python versions using _pyenv_. When I try to also set `system` in the `pyenv global ...` command I get the following error:

```
$ pyenv global system 3.5.4 3.4.7 3.3.6 3.2.6 2.7.14 2.6.9 pypy-5.7.1 pypy3.5-5.9.0 jython-2.7.1
pyenv: system version not found in PATH
```

I assume this is because of the missing `python` executable. Clearly, when I omit the `system` in the command above I have no access to the system Python anymore.

Provide A `python` Alias, Maybe?
----------
Is there a _pyenv_-supported way to use the system Python without dirty hacks? I know, I can add `system` on top of the `~/.pyenv/version` file manually, but adding a `python` alias pointing to the system `python3` may be nice, or is this not desirable?

**NOTE:** I didn't find a recommended way of providing a `python` executable alias on 17.10 again, neither a document that explains the decision why this was purposely omitted in 17.10. Maybe someone reading this can point to such a reading resource.

**Comment by maartjeth** [comment]
Have you already found a solution for this? I'm having similar issues.

**Comment by JustAnotherArchivist** [comment]
I have this issue now on a Debian Buster system with only Python 3 installed. Specifically, pyenv does not find the system `python3` executable and `pyenv versions` instead produces the warning "no Python detected on the system". (Cf. #1099; no, it is not a PATH issue.)

I think pyenv should check not [only for `python`](https://github.com/pyenv/pyenv/blob/610346802358780ebccafdf112ba0c92b6cbaef0/libexec/pyenv-versions#L105) but also for `python3`. Possibly also for `python2` (for systems where `python` defaults to Python 3 but only Python 2 is installed), but on the other hand, Python 2's *finally* EOL, so it should not be necessary to support such a setup anymore.

**Comment by xcdr** [comment]
I have the same problem on Ubuntu 20.04

**Comment by arkn98** [comment]
@xcdr Thanks for the workaround! But, I guess doing that is wrong, because `python2` and `python3` are not alternatives.

A better workaround is to install `python-is-python3` (maintained by the Ubuntu core developers; [link](https://packages.ubuntu.com/focal/python-is-python3)), which basically symlinks `/usr/bin/python` to `python3`. My `pyenv` properly detected the system version after installing this.

Additional info:
`pyenv --verion`: pyenv 1.2.18-7-gae4d4893
System python version: Python 3.8.2
Distro: Ubuntu 20.04 LTS

**Comment by native-api** [comment]
Should've been fixed by f77c2d7eab15d3ec04751c4545fabf5d5c56342b
